src/libraries/System.Runtime/tests/System.IO.FileSystem.Tests/Directory/GetFiles.cs

```diff
@@ -229,10 +229,10 @@ public void WindowsEnumerateDirectoryWithTrailingSpacePeriod(string dirName)
         {
             DirectoryInfo parentDir = Directory.CreateDirectory(GetTestFilePath());
             string problematicDirPath = Path.Combine(parentDir.FullName, dirName);
-            Directory.CreateDirectory(@"\\?\" + problematicDirPath);
```

Comment:
Preserving good fixes from closed PR: https://github.com/dotnet/runtime/pull/121038/files#diff-b577c61d28f3c8900172c55ad822d9dade5af6c3e059915d2d84bd68c26cb370R227 . (The test is still disabled.)